### ISSUE TITLE
Detroit: Become Human, PC version

### ISSUE THREAD

**Issue by MaxineAurora** [ogurets/popcnt_emulator#8]
So basically i've been using both **release** and **debug** builds following all of the instructions and tried to apply them to work with Detroit Become Human with a command like this:
`c:\bin\pin-2.14-71313-msvc11-windows\pin.exe -follow-execv -t64 instruction_hook64.dll -t instruction_hook32.dll -- DetroitBecomeHuman.exe`
But after i do this the game just gets stuck in Task manager forever with 0% CPU load in **release** build and 25% load in **debug** build and no error, is there some way to fix this?

**Comment by mirh** [comment]
Putting aside there should now be a fixed exe to bypass this requirement.. Do you have a core 2 duo?
And does plain SDE work?

**Comment by MaxineAurora** [comment]
plain SDE doesn't do anything still gives me an error, and i have 4 cores, not 2.

**Comment by mirh** [comment]
If even SDE isn't working there must be something sketchy going on behind. 
Are you using the latest version (that should be drm-free IIRC)?

**Comment by MaxineAurora** [comment]
Yeah, i'm using non-DRM version, free of Denuvo and all that stuff, basically here is the error that i'm getting in the link below (Note i do have SSE4.1 instructions, so error probably means only popcnt instruction, which is why i'm trying to use this emulator)
https://i.postimg.cc/vm2jHjr6/error-Detroit.png

**Comment by mirh** [comment]
Try to force sde to run with -snb or -skl switches?
It's very strange that it wouldn't work.

**Comment by MaxineAurora** [comment]
Nope that haven't helped at all

**Comment by mirh** [comment]
What if you try the demo?

**Comment by MaxineAurora** [comment]
I tried on both actually, demo and main game, it's the same.

**Comment by mirh** [comment]
Couldn't it be some antivirus crap?

**Comment by MaxineAurora** [comment]
nah

**Comment by mirh** [comment]
Sigh. 
Can you try in a clean VM?

**Comment by A1H1** [comment]
@MaxineAurora did you find any solution?

**Comment by xskorpiox** [comment]
> Yeah, i'm using non-DRM version, free of Denuvo and all that stuff, basically here is the error that i'm getting in the link below (Note i do have SSE4.1 instructions, so error probably means only popcnt instruction, which is why i'm trying to use this emulator)
> https://i.postimg.cc/vm2jHjr6/error-Detroit.png


It would be nice to indicate the processor model

**Comment by EvgeniySpinov** [comment]
I have Phenom II X6 1090T and I've played game with SDE - it gives more or less playable performance.

So it works with SDE with default settings for me. Probably the reason is within you processor or system.
